evaluator/src/main/kotlin/DependencyRule.kt

```diff
@@ -45,13 +45,13 @@ class DependencyRule(
     val dependency: PackageReference,
 
     /**
-     * The ancestors of the [dependency] in the dependency tree. The first entry is the root of the tree, the last entry
-     * (at [level] - 1) is the direct parent.
+     * The ancestors of the [dependency] in the dependency tree, sorted from farthest to closest: The first entry is the
+     * direct dependency of a project, the last entry (at index [level] - 1) is the direct parent of this [dependency].
      */
     val ancestors: List<PackageReference>,
 
     /**
-     * The level of the [dependency] inside the dependency tree. Starts with 0 for root level entries.
+     * The level of the [dependency] inside the dependency tree. Starts with 0 for the direct dependency of a project.
```

Comment:
Replace "the" with "a", otherwise it sounds like there can only be one direct dependency.